src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/AssertionHelper.java

```diff
@@ -42,14 +41,23 @@
 
 public class AssertionHelper {
 
+    public static Credential anEncryptingCredentialForTestRP() {
+        return new TestCredentialFactory(TEST_RP_PUBLIC_ENCRYPTION_CERT, TEST_RP_PRIVATE_ENCRYPTION_KEY).getEncryptingCredential();
+    }
+
     public static EncryptedAssertion anEidasEncryptedAssertion(String requestId, String issuerId, Signature assertionSignature) {
-        return anEidasEncryptedAssertion(requestId, issuerId, assertionSignature, anEidasAttributeStatement().build());
+        return anEidasEncryptedAssertionForTheTestRP(requestId, issuerId, assertionSignature, anEidasAttributeStatement().build());
```

Comment:
👍 to renaming this but I'd chop the `The`: `anEidasEncryptedAssertionForTestRP `